apphub/semantic_segmentation/unet3d/3DUnetMRBrainSegmentation_Isensee.ipynb

```diff
@@ -0,0 +1,1087 @@
+{
```

Comment:
why is the diceLoss code provided if it's all commented out? Likewise there's a commented out pipeline block at the end. Also, a spelling issue in first paragraph "fast estimator e**n**nvironment."

Reply:
DiceLoss code has been taken out.
For a memory constraint setup, the disabled pipeline code shows the pipeline setup. I would like to leave it there.

Reply:
If that's the point of the disabled pipeline can we put some helper text before the block to explain why it's there?

Reply:
Seprated the disabled pipeline into a cell with text explaining the expected usage scenario.